clang/test/CodeGenSYCL/loop_fusion_host.cpp

```diff
@@ -30,8 +29,3 @@ void foo() {
   kernel<class kernel_name_1>(f5);
 }
 // CHECK-NOT: !loop_fuse
-
-#if defined(DIAG)
```

Comment:
So what is happening here?  Are we suppressing errors as well?

Reply:
Yes, we're suppressing all checking. FWIW, I think this one probably should not be an error in any case (it seems like the sort of thing we can warn on and ignore the attribute because it's just an optimization hint and not going to cause miscompiles or anything. WDYT?

Reply:
I don't mind either way... we tend to default to errors on attributes for SYCL related attributes for some reason but that decision predates me paying attention here.

Reply:
For loop_fuse, semantic checking (without adding attribute) was done on host as well. This test checks for diagnostics on host. Since we decided to no longer do this, this  change is correct.